routers/routes/web.go

```diff
@@ -385,6 +385,7 @@ func RegisterRoutes(m *web.Route) {
 	m.Any("/user/events", reqSignIn, events.Events)
 
 	m.Group("/login/oauth", func() {
+		m.Get("/userinfo", user.UserInfoOAuth)
```

Comment:
It must be confirmed to user login

Reply:
Can you elaborate?

Reply:
`reqSignIn` will redirect to login page which is not suitable. An OAuth2 access token is required here.

Reply:
@lunny you can access this URL with an OAuth2 token, and a 302 redirect to login page would be treated as invalid from the oauth client.